src/sage/modular/multiple_zeta.py

```diff
@@ -380,7 +382,7 @@ def minimize_term(w, cf):
 
 # numerical values
 
-class MultizetaValues(UniqueRepresentation):
+class MultizetaValues(Singleton):
```

Comment:
Does this pickle correctly (does it have to pickle)?

Reply:
It does not make much sense to pickle that.